fill/fill_station.cc

```diff
@@ -126,34 +130,15 @@ MAIN
 // Start server and client services
 int main(int argc, char **argv)
 {
-<<<<<<< HEAD
-    // absl::ParseCommandLine(argc, argv);
-    // // Start the server in another thread
-    // std::shared_ptr<Server> server;
-    // RunServer(absl::GetFlag(FLAGS_server_port), server);
-    // // std::thread serverThread(RunServer, absl::GetFlag(FLAGS_server_port), server);
-=======
-    // begin the ADC
-    adc.begin();
+    WiringPiSetup(); 
```

Comment:
Also I figured out that WiringPiSetupGpio() uses the standard gpio pin numbers which is why we were getting issues. Please update this to use WiringPiSetupGpio() and update the stepper motor pins to the correct ones (GPIO numbering) on the Pi Hat confluence page